src/main/java/uk/gov/gchq/palisade/service/SimpleConnectionDetail.java

```diff
@@ -40,13 +41,15 @@ public SimpleConnectionDetail uri(final String uri) {
         return this;
```

Comment:
this mutator method needs @generated annotation adding

Reply:
Done